scripts/BUILD.bazel

```diff
@@ -163,3 +163,21 @@ kt_jvm_binary(
         "//scripts/src/java/org/oppia/android/scripts/todo:todo_check_lib",
     ],
 )
+
+kt_jvm_binary(
+    name = "closed_issue_check",
```

Comment:
This should be named specifically for TODOs (ditto for the class & Bazel target names).

Reply:
Done.